ospfd/ospf_vty.c

```diff
@@ -7023,6 +7029,21 @@ DEFUN (ip_ospf_area,
       return CMD_WARNING;
     }
 
+  // Check if we have an address arg and proccess it
+  if (argc == idx + 3) {
+    VTY_GET_IPV4_ADDRESS("interface address", addr, argv[(idx+2)]->arg);
```

Comment:
yep no need for additional checks.  The cli now ensures that the value entered is a legal v4 address.